python/mach_bootstrap.py

```diff
@@ -120,15 +133,22 @@ def _activate_virtualenv(topdir):
     virtualenv_path = os.path.join(topdir, "python", "_venv%d.%d" % (sys.version_info[0], sys.version_info[1]))
     python = sys.executable
 
-    if not in_venv():
-        venv_python_path = os.path.join(
-            virtualenv_path,
-            _get_virtualenv_script_dir(),
-            "python"
-        )
+    if os.environ.get("VIRUTAL_ENV") != virtualenv_path:
```

Comment:
typo: VIRUTAL_ENV → VIRTUAL_ENV

Reply:
Whoops. This is fixed in the latest branch.